aws/resource_aws_budgets_budget.go

```diff
@@ -343,6 +343,10 @@ func resourceAwsBudgetsBudgetNotificationRead(d *schema.ResourceData, meta inter
 
 	accountID, budgetName, err := decodeBudgetsBudgetID(d.Id())
 
+	if err != nil {
+		return err
```

Comment:
It should probably be smth more meaningful, like here: https://github.com/babbel/terraform-provider-aws/pull/4/files#diff-7b10e1e9bc5b2dcfb62b4a985832c22dR436

Reply:
@graffzon Yes thanks a lot. I changed it to include some context